src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -609,6 +612,14 @@ ucs_status_t uct_ib_mlx5_srq_init(uct_ib_mlx5_srq_t *srq, struct ibv_srq *verbs_
     srq->db  = srq_info.dv.dbrec;
     uct_ib_mlx5_srq_buff_init(srq, srq_info.dv.head, srq_info.dv.tail,
                               sg_byte_count, sge_num);
+
+    /* Initialize SRQ number */
+    dv.srq.in       = verbs_srq;
+    dv.srq.out      = &dvsrq;
+    dvsrq.comp_mask = MLX5DV_SRQ_MASK_SRQN;
+    mlx5dv_init_obj(&dv, MLX5DV_OBJ_SRQ);
```

Comment:
we already call mlx5dv_init_obj, line 589

Reply:
right, so should we add comp_mask before calling it in line 589

Reply:
maybe set comp_mask inside uct_ib_mlx5dv_init_obj, no need CHECK_DECLS

Reply:
but i think we wanted uct_ib_mlx5dv_init_obj behave as mlx5dv_init_obj and add only stuff which are missing from the original mlx5dv_init_obj

Reply:
also uct_ib_mlx5dv_init_obj is not used only for srq..